### PR TITLE
Some small changes

### PR DESCRIPTION
**Pull request by huixiancheng**
Remove invalid configuration. Moreover, look like it is necessary to use loss.item() instead of loss to avoid additional GPU memory consumption.

### CODE REVIEW

**Comment by huixiancheng** [comment]
A small writing error. Also, I would like to know why input_tensor=None, semantic_label=None, semantic_label_mask=None, semantic_output=None are set after each batch.
Also, it looks like you **did not set validation after the training epoch**, why is that? Also, does the **trainval** in code_mode  mean that the validation set is added to the training like 3Dmininet?https://github.com/placeforyiming/IROS21-FIDNet-SemanticKITTI/blob/825547433c490de0a12cc995811d43898b21a777/semantic_main.py#L28

**Comment by placeforyiming** [comment]
> A small writing error. Also, I would like to know why input_tensor=None, semantic_label=None, semantic_label_mask=None, semantic_output=None are set after each batch.
> Also, it looks like you **did not set validation after the training epoch**, why is that? Also, does the **trainval** in code_mode mean that the validation set is added to the training like 3Dmininet?
> 
> https://github.com/placeforyiming/IROS21-FIDNet-SemanticKITTI/blob/825547433c490de0a12cc995811d43898b21a777/semantic_main.py#L28

I set all those to None is because I observe memory leak sometimes, so just want to make sure manually release memory after each iteration.

For the validation, I save the checkpoint after each epoch and manually evaluate it. This is because I want to use the official evaluation script provided by the SemanticKITTI dataset, so I need to save all predicted labels locally, then run the script evaluation.sh.  The benefit is that the testing is for sure can work on the leaderboard as testing uses the same script with validation.

The **trainval** choice is what I tried in the early stage. If you keep digging, you may find I also have a script for multi-gpu training but didn't mention that in the readme file. The same thing happened for the learning rate scheduler, I tried many different training policies, but only present the best one I found.

I just strongly dislike the research style that adding complicated network modules with one or two points improvements. So this code aims to improve the network performance while keeping the network structure simple.  Most improvements happened on input, augmentation, learning rate scheduler, loss, etc. Our lab also works on hardware design, this structure is super friendly for hardware implementation

**Comment by huixiancheng** [comment]
Thank you for your reply. @placeforyiming  
Also, I totally agree with you.:thumbsup: However, the stress of graduation and life has prevented me from doing what I want to do and should do. :no_entry_sign:
